### PR TITLE
Update API version to v1.

### PR DESCRIPTION
**Pull request by Rohit-PX**
Signed-off-by: Rohit-PX <rohit@portworx.com>

**What type of PR is this?**
api-change

**What this PR does / why we need it**:
Update api versions from v1beta to v1 since v1beta will be deprecated

**Does this PR change a user-facing CRD or CLI?**:
no

**Is a release note needed?**:
no

**Does this change need to be cherry-picked to a release branch?**:
no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ provisioner: kubernetes.io/portworx-volume
 parameters:
    repl: "2"
 ---
-apiVersion: apps/v1beta1
+apiVersion: apps/v1
```
Comment (disrani-px) [inline README.md]: You'll also need the selector under spec.

Comment (Rohit-PX) [reply README.md]: Added selector for labels.

Comment (rodrigc) [reply README.md]: The selector you added is in the wrong place.  If I try to apply this yaml I get:

```
persistentvolumeclaim/mysql-data created
storageclass.storage.k8s.io/px-mysql-sc created
error: error validating "a.yml": error validating data: [ValidationError(Deployment.spec.template.spec): unknown field "selector" in io.k8s.api.core.v1.PodSpec, ValidationError(Deployment.spec): missing required field "selector" in io.k8s.api.apps.v1.DeploymentSpec]; if you choose to ignore these errors, turn validation off with --validate=false
```

Comment (rodrigc) [reply README.md]: You need to do something like this:

```
kind: PersistentVolumeClaim
apiVersion: v1
metadata:
   name: mysql-data
   annotations:
     volume.beta.kubernetes.io/storage-class: px-mysql-sc
spec:
   accessModes:
     - ReadWriteOnce
   resources:
     requests:
       storage: 2Gi
---
kind: StorageClass
apiVersion: storage.k8s.io/v1
metadata:
    name: px-mysql-sc
provisioner: kubernetes.io/portworx-volume
parameters:
   repl: "2"
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: mysql
spec:
  strategy:
    rollingUpdate:
      maxSurge: 1
      maxUnavailable: 1
    type: RollingUpdate
  replicas: 1
  selector:
    matchLabels:
      app: mysql  # needs to match spec.template.metadata.labels
  template:
    metadata:
      labels:
        app: mysql
        version: "1"
    spec:
      schedulerName: stork
      containers:
      - image: mysql:5.6
        name: mysql
        env:
        - name: MYSQL_ROOT_PASSWORD
          value: password
        ports:
        - containerPort: 3306
        volumeMounts:
        - name: mysql-persistent-storage
          mountPath: /var/lib/mysql
      volumes:
      - name: mysql-persistent-storage
        persistentVolumeClaim:
          claimName: mysql-data
```

Comment (rodrigc) [reply README.md]: See this example: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#components

The indentation is wrong in this example, but the comments give you an idea of what you need to do when adding a spec.selector

Comment (rodrigc) [reply README.md]: This is the same example, but with the indentation fixed:
https://github.com/kubernetes/website/blob/master/content/en/docs/concepts/workloads/controllers/statefulset.md#components

Comment (Rohit-PX) [reply README.md]: This has been fixed.

Comment (rodrigc) [reply README.md]: With the latest changes, this file now works with `kubectl apply`

File: specs/stork-daemonset.yaml [hunk 2]
```diff
@@ -83,7 +83,7 @@ spec:
       port: 8099
       targetPort: 8099
 ---
-apiVersion: extensions/v1beta1
```
Comment (rodrigc) [inline specs/stork-daemonset.yaml]: This file is missing a spec selector and does not apply:

```
error: error validating "stork-daemonset.yml": error validating data: ValidationError(DaemonSet.spec): missing required field "selector" in io.k8s.api.apps.v1.DaemonSetSpec
```

File: examples/groupvolumesnapshots/cassandra-e2e/cassandra-ss.yaml [hunk 3]
```diff
@@ -22,7 +22,7 @@ spec:
 
 ---
 
-apiVersion: "apps/v1beta1"
```
Comment (rodrigc) [inline examples/groupvolumesnapshots/cassandra-e2e/cassandra-ss.yaml]: This yaml needs a spec selector also:

```
error: error validating "cassandra-ss.yaml": error validating data: ValidationError(StatefulSet.spec): missing required field "selector" in io.k8s.api.apps.v1.StatefulSetSpec;
```

File: specs/stork-daemonset.yaml (hunk 2 above)
Comment (Rohit-PX) [reply specs/stork-daemonset.yaml]: Fixed.

File: examples/groupvolumesnapshots/cassandra-e2e/cassandra-ss.yaml (hunk 3 above)
Comment (Rohit-PX) [reply examples/groupvolumesnapshots/cassandra-e2e/cassandra-ss.yaml]: Fixed.

**Pull request merged**